### ISSUE TITLE
Go container node install fails with 'none' option

### ISSUE THREAD

**Issue by leighmcculloch** [microsoft/vscode-dev-containers#991]
<!--  🚨 Please only include bugs related to Dev Container Definitions here. 🚨 Other locations:
        VS Code Remote Development: http://github.com/Microsoft/vscode-remote-release 
        VS Code OSS: http://github.com/Microsoft/vscode
        GitHub Codespaces: https://github.com/github/feedback/discussions/categories/codespaces-feedback
-->

<!-- Please search existing issues to avoid creating duplicates. -->
<!-- Also please test using the latest insiders build to make sure your issue has not already been fixed: https://code.visualstudio.com/insiders/ -->

- VSCode Version: 1.59.0 (Universal)
- Local OS Version: macOS 11.5.1
- Reproduces in: Remote - Containers
- Name of Dev Container Definition with Issue: Go

Steps to Reproduce:

1. Create new container from repo such as [stellar/go](https://github.com/stellar/go)
2. Choose any Go version
3. Choose 'none' as Node version

```
[28158 ms] Start: Run: docker exec -t 5141405cc42f2acd8ec6fc56acd8759f1237d1ec39dfef9575242ce77093513a docker build -f /workspaces/go/.devcontainer/Dockerfile -t vsc-go-3273c0fb76b242d5edd0fad39c04b0d2 --build-arg VARIANT=1 --build-arg NODE_VERSION=none /workspaces/go/.devcontainer
[+] Building 6.9s (5/5) FINISHED                                                                                                                                                                             
 => [internal] load build definition from Dockerfile                                                                                                                                                    0.7s
 => => transferring dockerfile: 1.02kB                                                                                                                                                                  0.0s
 => [internal] load .dockerignore                                                                                                                                                                       0.8s
 => => transferring context: 2B                                                                                                                                                                         0.0s
 => [internal] load metadata for mcr.microsoft.com/vscode/devcontainers/go:0-1                                                                                                                          0.3s
 => CACHED [1/2] FROM mcr.microsoft.com/vscode/devcontainers/go:0-1@sha256:14d06147a1b7bfd34949c129778b899bcd566b6b28b5a6695ab8676d81bb3ce7                                                             0.0s
 => ERROR [2/2] RUN if [ "none" = "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install none 2>&1"; fi                                                                5.3s
------                                                                                                                                                                                                       
 > [2/2] RUN if [ "none" = "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install none 2>&1"; fi:
#5 3.320 Version 'none' not found - try `nvm ls-remote` to browse available versions.
------
executor failed running [/bin/sh -c if [ "${NODE_VERSION}" = "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi]: exit code: 3
```

**Comment by leighmcculloch** [comment]
It looks like #988 will fix this issue.

**Comment by Chuxel** [comment]
Fixed in #993 - Thanks for reporting!! There were two other spots as well, so good catch.

**Comment by Chuxel** [comment]
Workaround: Update the Dockerfile from...

```Dockerfile
RUN if [ "${NODE_VERSION}" = "none" ]; ...
```

to

```Dockerfile
RUN if [ "${NODE_VERSION}" != "none" ]; ...
```

or simply remove the `RUN` line if you don't want to install anything.
